packages/teleport/src/services/recordings/types.ts

```diff
@@ -10,12 +10,19 @@ export type RecordingsResponse = {
   startKey: string;
 };
 
+export type RecordingType = 'ssh' | 'desktop';
+export type RecordingDescription =
+  | 'play'
+  | 'recording disabled'
+  | 'non-interactive';
+
```

Comment:
why is it a `type` ? The type is a `string` and these are the values.

Reply:
I typed it because we are relying on this elsewhere to determine whether we show the "play" button or not ([link](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/Recordings/RecordingsList.tsx#L85-L91)). Awkward though, I'll use a constant instead.